server/src/infra/repositories/asset.repository.ts

```diff
@@ -184,10 +184,10 @@ export class AssetRepository implements IAssetRepository {
   }
 
   @GenerateSql({ params: [[DummyValue.UUID]] })
-  @ChunkedArray()
-  getByLibraryId(libraryIds: string[]): Promise<AssetEntity[]> {
-    return this.repository.find({
-      where: { library: { id: In(libraryIds) } },
+  getLibraryAssetPaths(pagination: PaginationOptions, libraryId: string): Paginated<AssetEntity> {
```

Comment:
Do you think it is worth defining a specific type so we know in the service there are only 3 fields on the "asset"?

Reply:
Yeah, that's a good idea